{"@t":"2024-02-02T11:40:55.5295930Z","@mt":"An error occurred: We have No orders with registerId: 12","@l":"Error","SourceContext":"Training.User.Middleware.ExceptionHandlingMiddleware","RequestId":"0HN13SBSUDG89:00000001","RequestPath":"/api/Order/GetOrders","ConnectionId":"0HN13SBSUDG89","ThreadId":25,"ProcessId":14376,"EnvironmentName":"Development","MachineName":"IN-PG0351Q9"}
{"@t":"2024-02-02T11:40:55.6053584Z","@mt":"An error occurred, StackTrace:    at Training.Order.Controllers.OrderController.GetOrders(Int32 registerId) in C:\\Users\\hariomroy\\Documents\\Test\\MircroServicesProject\\MySoln\\Training.Order\\Controllers\\OrderController.cs:line 44\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)\r\n   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)\r\n   at Training.User.Middleware.ExceptionHandlingMiddleware.InvokeAsync(HttpContext context, RequestDelegate next) in C:\\Users\\hariomroy\\Documents\\Test\\MircroServicesProject\\MySoln\\Training.Order\\Middleware\\ExceptionHandlingMiddleware.cs:line 23, Message: We have No orders with registerId: 12","@l":"Error","SourceContext":"Training.User.Middleware.ExceptionHandlingMiddleware","RequestId":"0HN13SBSUDG89:00000001","RequestPath":"/api/Order/GetOrders","ConnectionId":"0HN13SBSUDG89","ThreadId":25,"ProcessId":14376,"EnvironmentName":"Development","MachineName":"IN-PG0351Q9"}
